examples/local-config/external-network.yaml

```diff
@@ -19,7 +19,7 @@ spec:
     cloudName: openstack
     secretName: cloud-config
   managementPolicy: unmanaged
-  networkRef: external-network
   import:
     filter:
+      # FIXME(mandre) We don't have a way to reference the network by orc resource
```

Comment:
@mdbooth do you recall if this was the reason why we had `networkRef` in the spec directly?

Reply:
In other words, having the `networkRef` in the spec allows for imported resources to depend on other resources via their ORC names. The `cirros` example uses this behavior. This seems wrong. We should not add a dependency on other resources for unmanaged resources.